tools/generate_hotkeys.py

```diff
@@ -26,14 +35,33 @@
             "|Command|Key Combination|\n"
             "| :--- | :---: |\n"
         )
+        keys_list = []
         for help_text, key_combinations_list in categories[action].items():
+            keys_list.extend(key_combinations_list)
+            for key in key_combinations_list:
+                if key in KEYS_TO_EXCLUDE:
+                    keys_list.remove(key)
```

Comment:
This may be simpler with a comprehension?

Reply:
You can even bring it out of the for loop to do it just the once.

Reply:
Sure, I will bring that comprehension out of the for loop.